docs/userguides/networks.md

```diff
@@ -367,7 +367,15 @@ To configure network URIs in `node`, you can use the `ape-config.yaml` file:
 node:
   ethereum:
     mainnet:
-      uri: https://foo.node.bar
+      # For `uri`, you can use either HTTP, WS, or IPC values.
```

Comment:
I forgot about IPC! IPC actually supports `eth_subscribe` too (and it's faster/more reliable)